scripts/colab_install.py

```diff
@@ -15,13 +15,13 @@
 logger.setLevel(INFO)
```

Comment:
As a quick note, I think Google colab now requires python 3.7 to be specified for rdkit (https://stackoverflow.com/questions/63152656/installing-rdkit-in-google-colab). (Since Python 3.8 doesn't work with rdkit). Should we explicitly update the installer to handle this?

Reply:
> because of the recent Google colab update to python 3.8

I think the recent miniconda update to python 3.8. (not Google Colab, the python version of Google Colab is still 3.6.9)

> Should we explicitly update the installer to handle this?

I think we don't need. I didn't face the issue like stackoverflow by using the new command.

This script has installed python with the same version of Google Colab since before. Please see the following codes. 

https://github.com/deepchem/deepchem/blob/4fde71e354a71c5a2fa8b9f733e4abc31ae26a71/scripts/colab_install.py#L89
https://github.com/deepchem/deepchem/blob/4fde71e354a71c5a2fa8b9f733e4abc31ae26a71/scripts/colab_install.py#L120

Reply:
Ok sounds good!